packages/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -61,6 +74,7 @@ export class DropdownToggleCheckbox extends React.Component<DropdownToggleCheckb
           aria-invalid={!isValid}
           disabled={isDisabled}
           checked={this.calculateChecked()}
+          {...getOUIAProps(DropdownToggleCheckbox.displayName, ouiaId, ouiaSafe)}
```

Comment:
if you'd like better default id here also, for class based components the approach would be to add a state variable
```
import { getOUIAProps, OUIAProps, getDefaultOUIAId } from '../../helpers';
...
export class DropdownToggleCheckbox extends React.Component<DropdownToggleCheckboxProps, { ouiaStateId: string }> {
this.state = {
      ouiaStateId: getDefaultOUIAId(DropdownToggleCheckbox.displayName)
    };
}

return (
  <input {...getOUIAProps(DropdownToggleCheckbox.displayName, ouiaId !== undefined ? ouiaId : this.state.ouiaStateId, ouiaSafe)} />
)
```

Reply:
Done